test/factories/user.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
```

Comment:
For some reason `annotate` didn't wanna annotate the factories. It's supposed to Just Work™ but it didn't.